### PR TITLE
bug/custom_continue_button

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Allowed custom continue button to be applied to step nav view.
I also added the ability to change the progress view colors.  I thought it was going to be as simple as overriding the color extension's values but you cannot override class var for extensions like that unfortunately.

### CODE REVIEW

File: BridgeAppSDK/SBAStepProgressView.swift [hunk 1]
```diff
@@ -121,8 +126,8 @@ open class SBAStepProgressView: UIView {
         progressView.translatesAutoresizingMaskIntoConstraints = false
         stepCountLabel.translatesAutoresizingMaskIntoConstraints = false
         
-        backgroundView.backgroundColor = UIColor.headerViewProgressBackground
-        progressView.backgroundColor = UIColor.headerViewProgressBar
+        backgroundView.backgroundColor = SBAStepProgressView.progressBackgroundColor
+        progressView.backgroundColor = SBAStepProgressView.progressColor
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAStepProgressView.swift]: Subclassing and overriding will not return the progress bar background color. For that, you need to do this instead:

`type(of: self).progressBackgroundColor`

**Review by syoung-smallwisdom — CHANGES_REQUESTED** [review]

File: BridgeAppSDK/SBAStepProgressView.swift (hunk 1 above)
Comment (dephillipsmichael) [reply BridgeAppSDK/SBAStepProgressView.swift]: That's a good tip, will make the change

Comment (dephillipsmichael) [reply BridgeAppSDK/SBAStepProgressView.swift]: fixed

**Review by syoung-smallwisdom — APPROVED** [review]
looks good

**Pull request merged**